src/core/src/texture.rs

```diff
@@ -491,12 +491,16 @@ pub struct SamplerInfo {
     /// This bias is added to every computed mipmap level (N + lod_bias). For
     /// example, if it would select mipmap level 2 and lod_bias is 1, it will
     /// use mipmap level 3.
+    #[cfg_attr(feature = "serialize", serde(default = "SamplerInfo::def_lod_bias"))]
     pub lod_bias: Lod,
     /// This range is used to clamp LOD level used for sampling.
+    #[cfg_attr(feature = "serialize", serde(default = "SamplerInfo::def_lod_range"))]
     pub lod_range: (Lod, Lod),
     /// Comparison mode, used primary for a shadow map.
+    #[cfg_attr(feature = "serialize", serde(default = "None"))]
```

Comment:
this could be `serde(default)` without any mention of `None`, since it's a natural default

Reply:
Oh right, I've gotten so used to this annoying `= path` pattern I didn't notice ^^